src/FieldElementSubsystems.Test/UtilsTest.cs

```diff
@@ -0,0 +1,19 @@
+using EulynxLive.FieldElementSubsystems.Extensions;
+
+public class UtilsTest
+{
+    [Fact]
+    public static void TestHexToByteArray()
+    {
+        string hexString = "0A0B0C0D0E0F";

```

Comment:
I noticed that the test configuration contains a leading '0x' as the hex value. Should we have a test for that, too?